packages/orbit-components/src/Box/Box.stories.js

```diff
@@ -0,0 +1,272 @@
+      </Box>
+    );
+  })
+  .add("Positions", () => {
+    const position = select("position", POSITION, POSITION.RELATIVE);
+    const top = text("top", null);
+    const left = text("left", null);
+    const right = text("right", null);
+    const bottom = text("bottom", null);
+
+    return (
+      <Box
+        background={DEFAULT_COLOR}
+        height="full"
+        maxHeight="100px"
+        maxWidth="100px"
+        width="full"
+        top={top}
+        left={left}
+        right={right}
+        bottom={bottom}
+        position={position}
+      >
+        <div
+          style={{
+            height: "50px",
+          }}
+        />
+      </Box>
+    );
+  })
+  .add("Padding & Margin", () => {
+    const margin = select("margin", SPACINGS, SPACINGS[0]);
+    const padding = select("padding", SPACINGS, SPACINGS[0]);
+
+    return (
+      <Box background={DEFAULT_COLOR} maxWidth="300px" margin={margin} padding={padding}>
+        <div
+          style={{
+            height: "50px",
+          }}
+        />
+      </Box>
+    );
+  })
+  .add("BorderRadius", () => {
+    const radius = select("borderRadius", BORDER_RADIUS, BORDER_RADIUS.NORMAL);
+
+    return (
+      <Box background={DEFAULT_COLOR} maxWidth="150px" borderRadius={radius}>
+        <div
+          style={{
+            height: "50px",
+          }}
+        />
+      </Box>
+    );
+  })
+  .add("Colors", () => {
+    const color = select("colors", COLORS, COLORS[10]);
+    const background = select("background", COLORS, DEFAULT_COLOR);
+
+    return (
+      <Box display="flex" justify="center" align="center" background={background} color={color}>
+        <div
+          style={{
+            height: "100px",
+            width: "300px",
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
+        >
+          Content
+        </div>
+      </Box>
+    );
+  })
+  .add("Elevation", () => {
+    const elevation = select("elevation", ELEVATION, ELEVATION.ACTION);
+
+    return (
+      <Box
+        display="flex"
+        justify="center"
+        maxHeight="100px"
+        elevation={elevation}
+        maxWidth="300px"
+        align="center"
+        background="cloudDark"
+      >
+        <div
+          style={{
+            height: "100px",
+            display: "flex",
+            alignItems: "center",
+            justifyContent: "center",
+          }}
+        >
+          <Text>Content</Text>
+        </div>
+      </Box>
+    );
+  })
+  .add("Flex", () => {
+    const direction = select("direction", DIRECTION, DIRECTION.ROW);
+    const display = select("display", ["flex", "inline-flex"], "flex");
+    const align = select("align", ALIGN, ALIGN[0]);
+    const justify = select("justify", JUSTIFY, "start");
+    const wrap = select("wrap", WRAP, WRAP[0]);
+
+    return (
+      <Box
+        display={display}
+        justify={justify}
+        background="blueNormal"
```

Comment:
Would delete the background in this case and would change the background of the second example rectangle, otherwise is too confusing.